components/camel-jooq/src/main/java/org/apache/camel/component/jooq/JooqConfiguration.java

```diff
@@ -36,6 +34,9 @@
     @UriParam(label = "consumer", defaultValue = "true", description = "Delete entity after it is consumed.")
     private boolean consumeDelete = true;
 
+    @UriParam(description = "SQL query.")
+    private String query = "";
```

Comment:
I don't think the empty string is needed for initialization..

Reply:
Thanks, fixed.